main.go

```diff
@@ -67,39 +77,45 @@ func main() {
 	// prometheus.EnableCollectChecks(true)
 
 	r := httprouter.New()
-	r.Handler("GET", "/-/healthy", prometheus.InstrumentHandlerFunc("healthy", handler.Healthy(ms)))
-	r.Handler("GET", "/-/ready", prometheus.InstrumentHandlerFunc("ready", handler.Ready(ms)))
+	r.Handler("GET", *routePrefix+"/-/healthy", prometheus.InstrumentHandlerFunc("healthy", handler.Healthy(ms)))
+	r.Handler("GET", *routePrefix+"/-/ready", prometheus.InstrumentHandlerFunc("ready", handler.Ready(ms)))
 
-	r.Handler("GET", *metricsPath, prometheus.Handler())
+	r.Handler("GET", *routePrefix+*metricsPath, prometheus.Handler())
```

Comment:
If `metricsPath` doesn't have a leading `/`, and a `routePrefix` is set, this will result in something weird. Easy way out: Use https://golang.org/pkg/path/#Join

Reply:
🤦‍♂️ - I _had_ used `path.Join`, but I guess I didn't commit that 😉